Orange/tests/test_table.py

```diff
@@ -2195,6 +2195,12 @@ class _ExtendedTable(data.Table):
         self.assertIsInstance(data_file, _ExtendedTable)
         self.assertIsInstance(data_url, _ExtendedTable)
 
+    def test_str(self):
+        try:
```

Comment:
Fixed. Tbh I have no idea what's the real issue with the old code, it seems to be some complicated indexing issue back from when sparse matrices didn't support numpy style indexing.